src/cephadm/cephadm

```diff
@@ -5665,6 +5766,17 @@ def command_deploy(ctx):
                       reconfig=ctx.reconfig,
                       ports=daemon_ports)
 
+    elif daemon_type == CephExporter.daemon_type:
+        if not ctx.reconfig and not redeploy and not daemon_ports:
+            daemon_ports = list(CephExporter.port_map.values())
+
+        uid = 0
+        gid = 0
```

Comment:
any reason for the hardcoded 0/0 uid/gid rather than using `extract_uid_gid` like the other daemons? Does it absolutely need to be 0/0 for this to woek?

Reply:
Yeah for now I think it needs (0,0) , but I think as now the plan is to make it a ceph daemon (in the next iteration) it'll be easy to fetch the uid,gid just like other other ceph daemons